Index: database/migrations/2014_10_12_000000_create_users_table.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\nuse Illuminate\\Database\\Migrations\\Migration;\r\nuse Illuminate\\Database\\Schema\\Blueprint;\r\nuse Illuminate\\Support\\Facades\\Schema;\r\n\r\nclass CreateUsersTable extends Migration\r\n{\r\n    /**\r\n     * Run the migrations.\r\n     *\r\n     * @return void\r\n     */\r\n    public function up()\r\n    {\r\n        Schema::create('users', function (Blueprint $table) {\r\n            $table->id();\r\n            $table->string('name');\r\n            $table->string('email')->unique();\r\n            $table->timestamp('email_verified_at')->nullable();\r\n            $table->string('password');\r\n            $table->rememberToken();\r\n            $table->timestamps();\r\n        });\r\n    }\r\n\r\n    /**\r\n     * Reverse the migrations.\r\n     *\r\n     * @return void\r\n     */\r\n    public function down()\r\n    {\r\n        Schema::dropIfExists('users');\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- database/migrations/2014_10_12_000000_create_users_table.php	(revision 704d1a8b2cc304a31477701a3b6e335d4415dc8b)
+++ database/migrations/2014_10_12_000000_create_users_table.php	(date 1602538582771)
@@ -15,10 +15,13 @@
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('nombre');
+            $table->string('usuario');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
+            $table->string('telefono');
             $table->string('password');
+            $table->string('tipoUsuario');
             $table->rememberToken();
             $table->timestamps();
         });
